modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -51,17 +42,18 @@
 
 	subtle_message = span_emote("<b>[user]</b>[space]<i>[user.say_emphasis(subtle_message)]</i>")
 
-	for(var/mob/ghosts in GLOB.dead_mob_list)
-		if(!ghosts.client || isnewplayer(ghosts))
+	var/list/viewers = get_hearers_in_view(1, user)
+
+	for(var/mob/ghost in GLOB.dead_mob_list)
```

Comment:
```suggestion
	for(var/mob/ghost as anything in GLOB.dead_mob_list)
```

Reply:
Mmm, why the anything there? Will cause runtimes if the mob is somehow null or isn't *actually* a mob-

Reply:
it's a list of mobs, it can't not be a mob

Reply:
if there are non-mobs in the dead_mob_list we have bigger issues

Reply:
The reason we recommend using `as anything` is that BYOND won't run an implicit typecheck over each entry in that list, which means that it's going to iterate over them much faster. In this case, I think it'd be warranted for it to runtime if we have non-mobs in the `dead_mob_list`, because that's _absolutely_ not meant to happen.

Reply:
Ahhh, that makes sense, I'm just used to strictly typed everything, so-